nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/main/java/org/apache/nifi/groups/StandardProcessGroup.java

```diff
@@ -283,38 +283,21 @@ public boolean isRootGroup() {
     public void startProcessing() {
         readLock.lock();
         try {
-            for (final ProcessorNode node : processors.values()) {
+            findAllProcessors().stream().filter(SCHEDULABLE_PROCESSORS).forEach(node -> {
```

Comment:
Totally agree. There are a number of improvements that we can make with Java 8 available. Let's create a separate JIRA to identify those.